client/js/verification-portal/pregnancy-resource-center/get-resource-to-verify-button.js

```diff
@@ -12,10 +12,11 @@ const convertNumberToTimeFormat = timeNumber => {
 	if (!timeNumber) {
 		return null
 	}
-
 	const timeString = String(timeNumber).padStart(4, '0')
-
-	return `${timeString.slice(0, 2)}:${timeString.slice(2, 4)}`
+	const hours = parseInt(`${timeString.slice(0, 2)}`)
+	let am = ''
+	am = hours > 12 ? 'pm' : 'am'
```

Comment:
@bhavyaab This can be a confusing variable name because `am` could actually equal `'pm'`. The actual name is `meridiem`. What do you think of using that as a variable name instead?

Also, these 2 lines can be shortened to:
`const meridiem = hours > 12 ? 'pm' : 'am'`